luthier/config/src/config.lua

```diff
@@ -30,7 +30,14 @@ local tableConfig = tableUtils.build_table_config(tables)
 -- a script since creating a dir in Lua is awkard, in future.
 -- can be circumvented by using the LuaFileSystem module
 os.execute("mkdir -p ../external/")
+os.execute("mkdir -p ../../src/test/resource/")
 parser.save("../external/DimensionConfig.json", dimensionConfig)
 parser.save("../external/MetricConfig.json", metricConfig)
 parser.save("../external/TableConfig.json", tableConfig)
 -- parser.save("../external/MakerConfig.json", require("makers"))
```

Comment:
Why aren't we deleting all these lines? There's no point in
putting the JSON in two places.

Reply:
Done.